src/devices/UFireIse/README.md

```diff
@@ -0,0 +1,77 @@
+﻿# μFire ISE Probe Interface - I2C Driver
+
+## Summary
+
+The μFire ISE Probe Interface is an I²C sensor that can read a pH probe. Attach a waterproof temperature sensor for temperature compensation with the attached connector.
+
+## Usage
+
+You can find an example in the [sample](./samples/uFireIse.Sample.cs) directory. 
+
+
+### Basic
+It is possible to read the basic value (Electric Potential) from the probe.
+   
+```csharp
+using (UFireIse uFireIse = new UFireIse(device))
+{
+    Console.WriteLine("mV:" + uFireIse.Measure().Millivolts);
+}
+```
+
+### Orp 
+To read the ORP (OxidationReducationPotential) value use this example
+   
+```csharp
+using (UFireOrp uFireOrp = new UFireOrp(device))
+{
+	if (uFireOrp.TryMeasureOxidationReducationPotential(out ElectricPotential orp))
```

Comment:
should this be `ReductionPotential`? I assume you changed the code but sample is still left (you might want to search for Reducation and fix all)